### PR TITLE
Create first section of guest home page

### PR DESCRIPTION
**Pull request by DeLettenhove**
Create the welcome block of the guest landing page.

image

### CODE REVIEW

File: src/containers/guest-home-page/styles/WelcomeBlockStyles.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+        display: 'none'
+      }
+    },
+    'br:nth-of-type(2)': {
+      display: 'none',
+      [theme.breakpoints.down('md')]: {
+        display: 'flex'
+      },
+      [theme.breakpoints.down('sm')]: {
+        display: 'none'
```
Comment (tarasSanotskyi) [inline src/containers/guest-home-page/styles/WelcomeBlockStyles.js:38]: you can use breackpoints like this:
display: { xs: 'none', md: 'flex' }

File: src/containers/guest-home-page/styles/WelcomeBlockStyles.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+      br: {
+        display: 'none'
+      }
+    }
+  },
+  getStartBtn: {
+    display: 'block',
+    margin: '0px auto',
+    padding: '16px 51px',
+    color: '#fff',
```
Comment (tarasSanotskyi) [inline src/containers/guest-home-page/styles/WelcomeBlockStyles.js:95]: use variables fot white and black colors

Comment (DeLettenhove) [reply src/containers/guest-home-page/styles/WelcomeBlockStyles.js:95]: Fixed.

File: src/containers/guest-home-page/WelcomeBlock.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        { t('guestHomePage.welcomeBlock.subtitle.subtitlePart1') }
+        <br />
+        { t('guestHomePage.welcomeBlock.subtitle.subtitlePart2') }
+      </Typography >
+
+      <Button sx={ style.getStartBtn } >
+        <Typography >
+          { t('guestHomePage.welcomeBlock.btnText') }
+        </Typography >
+      </Button >
```
Comment (LubomyrKashuba) [inline src/containers/guest-home-page/WelcomeBlock.js:36]: Typography is useless inside the button, isn't it?

File: src/styles/app-theme/app.typography.js [hunk 4]
```diff
@@ -1,105 +1,89 @@
...
-    lineHeight: '30px'
+    fontSize: '24px',
+    lineHeight: '36px'
   },
   h6: {
-    color: palette.primary[900],
     fontWeight: 500,
     fontSize: '20px',
-    lineHeight: '28px',
+    lineHeight: '30px',
```
Comment (LubomyrKashuba) [inline src/styles/app-theme/app.typography.js:34]: image

lineHeight most be 28px

File: src/styles/app-theme/app.typography.js [hunk 5]
```diff
@@ -1,105 +1,89 @@
...
     textTransform: 'uppercase'
   },
   button: {
-    color: palette.primary[900],
     fontWeight: 500,
     fontSize: '16px',
     lineHeight: '24px',
     letterSpacing: '0.5px',
-    textTransform: 'capitalize'
+    textTransform: 'none'
```
Comment (LubomyrKashuba) [inline src/styles/app-theme/app.typography.js:79]: none is default for textTransform

File: src/styles/app-theme/app.typography.js (hunk 4 above)
Comment (DeLettenhove) [reply src/styles/app-theme/app.typography.js:34]: Fixed.

File: src/styles/app-theme/app.typography.js (hunk 5 above)
Comment (DeLettenhove) [reply src/styles/app-theme/app.typography.js:79]: Fixed.

File: src/containers/guest-home-page/WelcomeBlock.js (hunk 3 above)
Comment (DeLettenhove) [reply src/containers/guest-home-page/WelcomeBlock.js:36]: Yes, you are right, i fix this.

File: src/containers/guest-home-page/styles/WelcomeBlockStyles.js (hunk 1 above)
Comment (DeLettenhove) [reply src/containers/guest-home-page/styles/WelcomeBlockStyles.js:38]: Fixed.

File: public/index.html [hunk 6]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
+<html lang="en" >
```
Comment (maarekua) [inline public/index.html:2]: Do we need to edit this file?

File: src/constants/translations/en/guest-home-page.json [hunk 7]
```diff
@@ -0,0 +1,6 @@
+{
+  "welcomeBlock": {
+    "subtitle": "An educational platform that offers professional courses in It is a long established fact that a reader will be distracted by the readable content of",
+    "btnText": "Get started for free"
```
Comment (maarekua) [inline src/constants/translations/en/guest-home-page.json:4]: not btnText its getStarted

Comment (maarekua) [inline src/constants/translations/en/guest-home-page.json:4]: not btnText its getStarted

File: src/constants/translations/en/guest-home-page.json [hunk 8]
```diff
@@ -0,0 +1,6 @@
+{
+  "welcomeBlock": {
+    "subtitle": "An educational platform that offers professional courses in It is a long established fact that a reader will be distracted by the readable content of",
```
Comment (maarekua) [inline src/constants/translations/en/guest-home-page.json:3]: description

Comment (maarekua) [inline src/constants/translations/en/guest-home-page.json:3]: description

File: src/constants/translations/ua/guest-home-page.json [hunk 9]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (maarekua) [inline src/constants/translations/ua/guest-home-page.json:1]: First of all, it's Ukrainian, and do we need it right now at all? I don't think so.

Comment (maarekua) [inline src/constants/translations/ua/guest-home-page.json:1]: First of all, it's Ukrainian, and do we need it right now at all? I don't think so.

File: src/containers/guest-home-page/styles/WelcomeBlockStyles.js [hunk 10]
```diff
@@ -0,0 +1,72 @@
+import theme from '~/styles/app-theme/custom-mui.styles'
+import desktopBg from '~/img/guest-home-page/desktopBg.svg'
+import tabletBg from '~/img/guest-home-page/tabletBg.svg'
+import mobileBg from '~/img/guest-home-page/mobileBg.svg'
+import titleMd from '~/img/guest-home-page/titleMd.svg'
+import titleSm from '~/img/guest-home-page/titleSm.svg'
+import titleXs from '~/img/guest-home-page/titleXs.svg'
+
+export const WelcomeBlockStyles = {
```
Comment (maarekua) [inline src/containers/guest-home-page/styles/WelcomeBlockStyles.js:9]: Clean the CSS, and rename it with camelCase style. It is not a component.

Comment (maarekua) [inline src/containers/guest-home-page/styles/WelcomeBlockStyles.js:9]: Clean the CSS, and rename it with camelCase style. It is not a component.

File: src/containers/guest-home-page/styles/WelcomeBlockStyles.js [hunk 11]
```diff
@@ -0,0 +1,72 @@
+import theme from '~/styles/app-theme/custom-mui.styles'
+import desktopBg from '~/img/guest-home-page/desktopBg.svg'
```
Comment (maarekua) [inline src/containers/guest-home-page/styles/WelcomeBlockStyles.js:2]: All images should be moved to assets/images folder.

Comment (maarekua) [inline src/containers/guest-home-page/styles/WelcomeBlockStyles.js:2]: All images should be moved to assets/images folder.

**Review by maarekua — CHANGES_REQUESTED** [review]

File: src/constants/translations/en/guest-home-page.json (hunk 7 above)
Comment (DeLettenhove) [reply src/constants/translations/en/guest-home-page.json:4]: Fixed.

Comment (DeLettenhove) [reply src/constants/translations/en/guest-home-page.json:4]: Fixed.

File: src/constants/translations/en/guest-home-page.json (hunk 8 above)
Comment (DeLettenhove) [reply src/constants/translations/en/guest-home-page.json:3]: Fixed.

File: src/constants/translations/ua/guest-home-page.json (hunk 9 above)
Comment (DeLettenhove) [reply src/constants/translations/ua/guest-home-page.json:1]: Fixed.

Comment (DeLettenhove) [reply src/constants/translations/ua/guest-home-page.json:1]: Fixed.

File: src/containers/guest-home-page/styles/WelcomeBlockStyles.js (hunk 11 above)
Comment (DeLettenhove) [reply src/containers/guest-home-page/styles/WelcomeBlockStyles.js:2]: Fixed.

**Review by maarekua — APPROVED** [review]

**Pull request merged**